applications/DEMApplication/python_scripts/analytic_tools/analytic_data_procedures.py

```diff
@@ -88,17 +104,34 @@ def __init__(self, model_part, main_path, do_clear_data = True):
         self.name_avg_vel_nr = 'mass_avg_normal_vel'
         self.ghost_smp_detected = False
 
-        for sub_part in self.model_part:
-            if sub_part[IS_GHOST] == True:
+        for smp in sub_model_parts:
+            if smp[IS_GHOST] == True:
                 self.ghost_smp_detected = True
-                self.face_watcher_dict[sub_part.Name] = AnalyticFaceWatcher(sub_part)
-                self.face_watcher_analysers[sub_part.Name] = FaceWatcherAnalyzer(name=sub_part.Name, analytic_face_watcher=self.face_watcher_dict[sub_part.Name])
+                self.surface_analyzers_list.append(SurfaceAnalyzer(smp))
 
+        self.surface_analyzers_list_size = len(self.surface_analyzers_list)
         self.RemoveFiles()
 
+    #TODO:remove b4 merge
+    def original_MakeAnalyticsMeasurements(self):
+        for obj in self.surface_dict.values():
+            obj.MakeMeasurements()
+
+    #TODO:remove b4 merge
```

Comment:
what does this mean?

Reply:
previous implementation for testing purposes.